### PR TITLE
Fixes: https://github.com/rust-vmm/rust-vmm-ci/issues/8

### PR DESCRIPTION
**Pull request by anth1y**
cc/ @andreeaflorescu @sameo @lauralt

### CODE REVIEW

**Comment by andreeaflorescu** [comment]
@anth1y can you please sign your commit:
```git commit --amend -s --no-edit``` and then do a `push -f`.

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -203,3 +203,16 @@ steps:
           image: "rustvmm/dev:v6"
           always-pull: true
           propagate-environment: true
+
+- label: "cargo-audit"
+     commands:
+       - cargo audit
```
Comment (lauralt) [inline .buildkite/pipeline.yml:209]:
```suggestion
       - cargo audit -q
```
This is what I was suggesting in #36 (sorry if it wasn't clear enough). Do you have any concerns about this change?

**Comment by anth1y** [comment]
@andreeaflorescu my commit is signed if you click on the `verified` tag you'll see my gpg key

**Comment by anth1y** [comment]
I just realized the gpg signage is only displayed in the web ui 🤦

**Review by lauralt — APPROVED** [review]

**Review by andreeaflorescu — APPROVED** [review]

**Pull request merged**